### ISSUE TITLE
 A "PlaybackStopEvent" should be fired on track changes

### ISSUE THREAD

**Issue by Dperrigo18** [dkanada/gelli#143]
This is needed for last.fm scrobbling to work properly.

Here is a detailed description of the last.fm plugin behavior:
https://github.com/jesseward/jellyfin-plugin-lastfm/issues/27#issuecomment-744031810

**Comment by Saladaa** [comment]
Hi, I think it would be handy to also have Finamp's thread linked in case any information is relevant there if someone decides to work on this (supposedly Finamp will have this fixed in a release build soon):
https://github.com/UnicornsOnLSD/finamp/issues/87

This issue also effects the ListenBrainz plugin (which I personally use) in (probably) the same way: https://github.com/lyarenei/jellyfin-plugin-listenbrainz
The ListenBrainz plugin was originally just a fork of the Last FM plugin but it has diverged to become its own thing now.

**Comment by lyarenei** [comment]
> This issue also effects the ListenBrainz plugin (which I personally use) in (probably) the same way
> The ListenBrainz plugin was originally just a fork of the Last FM plugin but it has diverged to become its own thing now.

Yes, that is correct. Both plugins use the same approach to decide whether to send a scrobble or not, there was no reason to change it. The plugins only differ in the features as they support different services.
